vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# vars file for arillso.logrotate
+hourly_link_state: absent
```

Comment:
This is obsolete in my proposal.

Reply:
Fix in 17cc47173ce9945c74dd3629f6912df504a8eeeb